src/styles/tokens.scss

```diff
@@ -0,0 +1,187 @@
+@use 'sass:string';
+@use 'sass:map';
+@use 'sass:list';
+
+/* Constants */
+$tail-sizes: (xs, sm, md, lg, xl, 2xl);
+$tail-sizes-with-badge: (md, lg, xl, 2xl);
+$tail-sizes-with-href: (md, lg, xl, 2xl);
+$tail-group-sizes: (xs, sm, md, lg);
+$tail-variants: (as, ha, oa);
+
+/* Naming Convention:
+ * - "container": The circular container/background that holds the tail icon
+ * - "tail": The actual airline livery SVG icon displayed within the container
+ */
+
+/* Size Maps */
+$container-width: (
+  'xs': 24px,
+  'sm': 32px,
+  'md': 48px,
+  'lg': 60px,
+  'xl': 96px,
+  '2xl': 134px
+);
+
+$tail-width: (
+  'xs': 19px,
+  'sm': 26px,
+  'md': 39px,
+  'lg': 48px,
+  'xl': 77px,
+  '2xl': 107px
+);
+
+/* Adjust vertical offset to visually center tail icon within circular container
+ * Based on optical centering from design spec, not mathematical centering */
+$tail-offset-y: (
+  'xs': 3px,
+  'sm': 4px,
+  'md': 6px,
+  'lg': 7px,
+  'xl': 13px,
+  '2xl': 17px
+);
+
+$href-label-gap: (
+  'md': 2px,
+  'lg': 4px,
+  'xl': 6px,
+  '2xl': 8px
+);
+
+$badge-width: (
+  'md': 11px,
+  'lg': 14px,
+  'xl': 22px,
+  '2xl': 31px
+);
+
+$badge-offset: (
+  'md': (x: 2px, y: -7px),
+  'lg': (x: 2px, y: -4px)
+);
+
+$group-diagonal-width: (
+  'xs': 42px,
+  'sm': 56px,
+  'md': 84px,
+  'lg': 105px
+);
+
+/* Group - Horizontal Map */
+$group-horizontal: (
+  'lg': (border-width: 4px, overlap: -10px),
+  'md': (border-width: 4px, overlap: -6px),
+  'sm': (border-width: 3px, overlap: -2px),
+  'xs': (border-width: 3px, overlap: -2px)
+);
+
+/* Token Definitions */
+:host {
+  display: inline-block;
+  /* Host dimensions to match container */
+  width: var(--container-width);
+  height: var(--container-width);
+  min-width: var(--container-width);
+  min-height: var(--container-width);
+
+  /* Default Values */
+  --container-width: var(--container-width-lg);
+  --container-gradient: var(--gradient-oa);
+  --tail-width: var(--tail-width-lg);
+  --tail-offset-y: var(--tail-offset-y-lg);
+  --href-label-gap: var(--href-label-gap-lg);
+  --badge-width: var(--badge-width-lg);
+  
+  /* Border defaults */
+  --border-color-default: rgba(88, 94, 103, 1);
+  --border-color-white: #ffffff;
```

Comment:
We need to be using design tokens everywhere possible in our own styles, both color and size